api/tests/unit/application/certification-point-of-contacts/certification-point-of-contact-controller_test.js

```diff
@@ -45,6 +45,7 @@ describe('Unit | Controller | certifications-point-of-contact-controller', funct
             'first-name': 'Buffy',
             'last-name': 'Summers',
             email: 'buffy.summers@example.net',
+            lang: 'fr',
```

Comment:
Suggestion : je ne sais plus qui suggérait sur slack je crois ou en tech time de tester le retour de l'api en acceptance en plus du code retour.
ici ça serait dans `api/tests/acceptance/application/certification-point-of-contacts/certification-point-of-contacts_test.js` on pourrait ajouter cette assertion 
`expect(response.result.data.attributes.lang).to.equal('fr');`

Reply:
Fait. Le test d'acceptance a été modifié comme suggéré. Merci !